BUILD

```diff
@@ -4,19 +4,21 @@ objc_library(
     name = "MOLXPCConnection",
     srcs = ["Source/MOLXPCConnection/MOLXPCConnection.m"],
     hdrs = ["Source/MOLXPCConnection/MOLXPCConnection.h"],
+    copts = ["-Wunguarded-availability"],
```

Comment:
Can you add this to the podspec too?

`s.compiler_flags = '-Wunguarded-availability'`

Should do it